### PR TITLE
Adding blog post about HaxeFlixel Mechanics

### PR DESCRIPTION
**Pull request by SeiferTim**

### CODE REVIEW

File: src/documents/blog/07-haxeflixel-mechanics.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Well, what if I told you that it *already exists*!?
+
+A couple of years back now (wow, time flies!), I had the idea of putting together a website that would showcase a ton of little samples of HaxeFlixel code, along with a demonstration of that code in action. It took some time, but I was able to get a pretty good start on this thing, and I called it *HaxeFlixel Mechanics*. The plan was to get a bunch of samples in the project, and then invite collaborators to add more to help grow and flesh it out... and then Real Life&trade; got in the way and the project got pushed aside...
+
+This year, let's work together to get HFMechanics back up and running again. Let's add more demoes and snippets, and try to fill it with lots more good stuff.
+
+If you'd like to lend a hand, join us over in (the repository)[https://github.com/HaxeFlixel/haxeflixel-mechanics]. Add an issue for suggestions or requests, or add a PR for snippets you've created so we can add them to the site!
```
Comment (Gama11) [inline src/documents/blog/07-haxeflixel-mechanics.md]: You got the links wrong, `[` and `(` brackets need to be flipped I think.

**Comment by Gama11** [comment]
This needs _at the very least_ one picture of the HF-mechanics site, maybe even an embedded swf? :)

**Comment by Gama11** [comment]
You can put pictures into `images/blog/`.

File: src/documents/blog/07-haxeflixel-mechanics.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+If you want to check out the site in action, you can visit (http://hfmechanics.nfshost.com/)[http://hfmechanics.nfshost.com/] for a live version!
```
Comment (Gama11) [inline src/documents/blog/07-haxeflixel-mechanics.md]: I think this link needs to be earlier, probably at the first mention of "HaxeFlixel Mechanics".

File: src/documents/blog/07-haxeflixel-mechanics.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+Well, what if I told you that it *already exists*!?
+
+A couple of years back now (wow, time flies!), I had the idea of putting together a website that would showcase a ton of little samples of HaxeFlixel code, along with a demonstration of that code in action. It took some time, but I was able to get a pretty good start on this thing, and I called it *HaxeFlixel Mechanics*. The plan was to get a bunch of samples in the project, and then invite collaborators to add more to help grow and flesh it out... and then Real Life&trade; got in the way and the project got pushed aside...
+
+This year, let's work together to get HFMechanics back up and running again. Let's add more demoes and snippets, and try to fill it with lots more good stuff.
```
Comment (Gama11) [inline src/documents/blog/07-haxeflixel-mechanics.md]: I don't think "demos" is spelled with that additional "e"?

File: src/documents/blog/07-haxeflixel-mechanics.md (hunk 1 above)
Comment (Gama11) [inline src/documents/blog/07-haxeflixel-mechanics.md]: I think "submit" a PR sounds better than "add". Also, I wouldn't use the abbreviation, some people might not be familiar with it.

Comment (Gama11) [inline src/documents/blog/07-haxeflixel-mechanics.md]: Also, you should _really_ link to the Contributing section of the wiki here. :)

https://github.com/HaxeFlixel/haxeflixel-mechanics/wiki/Contributing

File: src/documents/blog/07-haxeflixel-mechanics.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+title: "HaxeFlixel Mechanics"
+layout: blog-post
+postDate: 03-02-2017
+```
+
+Does this sound familiar? "I wish I could remember how to do this one, small thing in HaxeFlixel, and that I didn't have to dig through a massive tutorial just to jog my memory." Or, how about: "What's the 'right' way to handle this, again?" Yeah, we've all been there! Wouldn't it be nice if there was some kind of interactive repository of just about every conceiveable useful snippet of code that you could use to answer these, and other questions?
+
+Well, what if I told you that it *already exists*!?
+
+A couple of years back now (wow, time flies!), I had the idea of putting together a website that would showcase a ton of little samples of HaxeFlixel code, along with a demonstration of that code in action. It took some time, but I was able to get a pretty good start on this thing, and I called it *HaxeFlixel Mechanics*. The plan was to get a bunch of samples in the project, and then invite collaborators to add more to help grow and flesh it out... and then Real Life&trade; got in the way and the project got pushed aside...
```
Comment (Gama11) [inline src/documents/blog/07-haxeflixel-mechanics.md]: You say "I" here, and it's fine to write it from your perspective, but the reader of the post has no idea who the author is. You should introduce yourself somehow. :)

File: src/documents/blog/07-haxeflixel-mechanics.md (hunk 3 above)
Comment (SeiferTim) [inline src/documents/blog/07-haxeflixel-mechanics.md]: Nope, I'm 100% sure on this one!

image

Comment (SeiferTim) [inline src/documents/blog/07-haxeflixel-mechanics.md]: Oops, yeah, I KNOW that, too, dunno howthat slipped in there...

**Pull request merged**